SparkPost/2web2ui
# SA-1600: Added check for subaccount to include the grid column

### What Changed
 - Removed empty subaccount select column on signals pages if there are no subaccounts

### How To Test
 - Create new account
 - Go to any of the signals pages
 - Verify the breakdown select is on the left instead of spaced in the middle

### To Do
